sequencer/dbmanager.go

```diff
@@ -178,18 +195,50 @@ func (d *dbManager) GetWIPBatch(ctx context.Context) (*WipBatch, error) {
 	}
 
 	wipBatch := &WipBatch{
-		batchNumber:  lastBatch.BatchNumber,
-		coinbase:     lastBatch.Coinbase,
-		accInputHash: lastBatch.AccInputHash,
-		// initialStateRoot: lastBatch.StateRoot,
+		batchNumber:    lastBatch.BatchNumber,
+		coinbase:       lastBatch.Coinbase,
+		accInputHash:   lastBatch.AccInputHash,
```

Comment:
We need to get also `initialAccInputHash` by checking if there was a previous closed batch to get I t from it otherwise we use the `lastBatch.AccInputHash`